web-app/src/pages/Login/LoginPage.tsx

```diff
@@ -84,9 +106,10 @@ export default function AuthComponent() {
 		}));
 	}, [authData.username, authData.password, authData.confirmedPassword]);
 
-	const handleLogin = () => {
-		//TODO Implements back auth routes
-		if (authData.username === "b" && authData.password === "b") {
+	const handleLogin = async () => {
+
```

Comment:
Why is there 2 empty lines?

Reply:
## Fix

Identation is fixed with Prettier.